docs/Architecture-Overview.md

```diff
@@ -20,6 +20,9 @@ specification (https://www.cbioportal.org/api/). Note that the repo where this
 lives in (https://github.com/cBioPortal/cbioportal) also contains Java classes
 to import data as well as the validator.
 
+The backend is organized as a multi-module Maven project.
+See [cBioPortal code organization](Code-Organization.md).
```

Comment:
I first thought having a single integrated source of information/docs was a good idea, but now I think it may be too much information.  The fact that we have two repos hints that the documentation can be separate, but we should link the repos together at the appropriate places.

So, rename this to "cBioPortal backend code organization" ?

Reply:
renamed to Backend-Code-Organization.md